estate/models/estate_property.py

```diff
@@ -0,0 +1,41 @@
+"""module for the estate property model"""
+
+from odoo import models, fields, api
+
+class EstateProperty(models.Model):
+    "Estate property odoo model"
+    _name = "estate.property"
+    _description= "model for real estate assets"
+    #
+    name = fields.Char("Name", required = True)
+    description = fields.Text("Description")
+    postcode = fields.Char("Postcode")
+    date_availability = fields.Date("Available From", default = lambda _: fields.Date.add(fields.Date.today(), months=3))
+    expected_price = fields.Float("Expected Price", required = True)
+    selling_price = fields.Float("Selling Price")
+    bedrooms = fields.Integer("# bedrooms", default = 2)
+    living_area = fields.Integer("Living Area (m2)")
+    facades = fields.Integer("# facades")
+    garage = fields.Boolean("Has Garage")
+    garden = fields.Boolean("Has Garden")
+    garden_area = fields.Integer("Garden Area (m2)")
+    garden_orientation = fields.Selection([ ("north", "North"), ("south", "South"), ("east", "East"), ("west", "West") ])
+    active = fields.Boolean(default = True)
+    state = fields.Selection([ ("new", "New"), ("offer_received", "Offer Received"), ("offer_accepted", "Offer Accepted"), ("sold", "Sold"), ("canceled", "Canceled") ], required = True, default = "new", string = "Bar", copy=False)
+    create_date = fields.Datetime(copy = False)
```

Comment:
This field already exists on your model because you inherit from `models.Model`, no need to redeclare it here